### PR TITLE
#mission_nps_mobile_40: Prompt user to use bulkfyle after using instafyle thrice

### PR DESCRIPTION
**Pull request by yksolanki9**
UI when camera opened for the 4th time:
https://user-images.githubusercontent.com/39493102/200512391-f8485e76-b742-46fd-bd0a-edcf33a49404.mp4

UI Preview:
Screenshot 2022-11-08 at 1 12 52 PM

### CODE REVIEW

File: src/app/core/services/transactions-outbox.service.ts [hunk 1]
```diff
@@ -32,6 +32,9 @@ export class TransactionsOutboxService {
 
   ROOT_ENDPOINT: string;
 
+  //Used for showing bulk mode prompt when instafyle is used more than thrice in the same session
+  singleCaptureCount = 0;
```
Comment (Julias0) [inline src/app/core/services/transactions-outbox.service.ts:36]: make this variable private and allow access throw methods only. Giving variable access inside services is an antipattern

Comment (Julias0) [reply src/app/core/services/transactions-outbox.service.ts:36]: getSingleCaptureCountInSameSession
setSingleCaptureCountInSameSession

Comment (Julias0) [reply src/app/core/services/transactions-outbox.service.ts:36]: Also attach a video recording for me to review the interaction

**Review by Julias0 — CHANGES_REQUESTED** [review]
Added comments

File: src/app/core/services/transactions-outbox.service.ts (hunk 1 above)
Comment (yksolanki9) [reply src/app/core/services/transactions-outbox.service.ts]: Have added a getter method and another method to increment the count.
Have also attached a recording - Sorry for this

**Review by Julias0 — APPROVED** [review]

**Pull request merged**